spec/features/user_register_and_pay_with_card_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+feature 'user register and pay with card', {js: true, vcr: true} do 
+
+  background do 
+    visit register_path
+  end
+  scenario 'with valid user info and credit card' do 
+    fill_in_form('password','4242424242424242')
+    page.should have_content 'You are successful Registed and Signed in'
+  end
+  scenario 'with valid user info and invalid card' do 
+    fill_in_form('password','123')
+    page.should have_content 'This card number looks invalid.'
+  end
+  scenario 'with valid user info and declined card' do 
+    fill_in_form('password','4000000000000002')
+    page.should have_content 'Your card was declined.'
+  end
+  scenario 'with invalid user info and valid card' do 
+    fill_in_form(nil,'4242424242424242')
+    page.should have_content "can't be blank"
+  end
+  scenario 'with invalid user info and invalid card' do 
+    fill_in_form(nil,'123')
+    page.should have_content 'This card number looks invalid.'
+  end
+  scenario 'with invalid user info and declined card' do 
+    fill_in_form(nil,'4000000000000002')
+    page.should have_content "can't be blank"
```

Comment:
You are using old `should` syntax, but in other RSpec files you are using `expect` syntax. I would recommend to stick to one style - preferably `expect` as its newer and current way.